packages/webapp/graphql/ssr.client.ts

```diff
@@ -0,0 +1,4 @@
+import { GraphQLClient } from 'graphql-request';
+
+// this hits the graphql api exposed in the pages/api/graphql.ts file
```

Comment:
Don't think this comment is necessary tbh, it's sort of self commenting. I also do not like comments that reference specific implementation details, in this case the path of the API, because it's very easy for them to go out of date unnoticed and cause more confusion than they relieve in the future.